syntax="proto2";

package aap_protobuf.service.media.sink;

enum MediaMessageId
{
    MEDIA_MESSAGE_DATA = 0;
    MEDIA_MESSAGE_CODEC_CONFIG = 1;
    MEDIA_MESSAGE_SETUP = 32768;
    MEDIA_MESSAGE_START = 32769;
    MEDIA_MESSAGE_STOP = 32770;
    MEDIA_MESSAGE_CONFIG = 32771;
    MEDIA_MESSAGE_ACK = 32772;
    MEDIA_MESSAGE_MICROPHONE_REQUEST = 32773;
    MEDIA_MESSAGE_MICROPHONE_RESPONSE = 32774;
    MEDIA_MESSAGE_VIDEO_FOCUS_REQUEST = 32775;
    MEDIA_MESSAGE_VIDEO_FOCUS_NOTIFICATION = 32776;
    MEDIA_MESSAGE_UPDATE_UI_CONFIG_REQUEST = 32777;
    MEDIA_MESSAGE_UPDATE_UI_CONFIG_REPLY = 32778;
    MEDIA_MESSAGE_AUDIO_UNDERFLOW_NOTIFICATION = 32779;
}
